content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -319,21 +319,24 @@ Some values of an object are typically generated before the object is persisted.
 
 ## Server Side Apply
 
-Server Side Apply (SSA) is offering an apply path on the apiserver side of kubernetes instead of the previous/current Client Side Apply (CSA) that happens in kubectl.
+{{< feature-state for_k8s_version="v1.14" state="alpha" >}} In vesion 1.14, if the Server Side Apply feature is enabled, The `PATCH` endpoint will accept the additional `application/apply-patch+yaml` content type. Server Side Apply allows clients other than kubectl to perform the Apply operation, and will eventually fully replace the complicated Client Side Apply logic that only exists in kubectl.
```

Comment:
> In vesion 1.14
:-)

Reply:
I find it surprising that this section starts with a super technical explanation? Should we at least introduce somehow?

Reply:
> Server Side Apply allows clients other than kubectl to perform the Apply operation, and will eventually fully replace the complicated Client Side Apply logic that only exists in kubectl.

Maybe a variation of this sentence should go first?

Reply:
Sure I agree.